### ISSUE TITLE
Minimum Area shape  requirement could use attention, Focus Appearance (Minimum) 

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#1928]
The second bullet of 2.4.11 Focus Appearance (Minimum) reads:
> Minimum area: The contrasting area is at least as large as:
>- Outline: the area of a 1 CSS pixel thick perimeter of the unfocused component, or
>- Shape: the area of a 4 CSS pixel thick line along the shortest side of a minimum bounding box of the unfocused component, and no thinner than 2 CSS pixels.

I find the phrase “and no thinner than 2 CSS pixels” in the shape sub-bullet to be a little awkward, especially as it follows a discussion of area. it might help to reverse the order to make it clearer the thinness refers to the shape and not the area:

>- Shape: a shape no thinner than 2 CSS pixels, covering an area equivalent to a 4 CSS pixel thick line along the shortest side of a minimum bounding box of the unfocused component.

**Comment by alastc** [comment]
Hi @mbgower,

To me saying that "a share no thinner than 2 CSS pixels" implies no part of it could be thinner. 

How about this which I proposed for #1806 

> Shape: the area of a 4 CSS pixel thick line along the shortest side of a <a>minimum bounding box</a> of the unfocused component, and **no part of the contributing area** is thinner than 2 CSS pixels

**Comment by mraccess77** [comment]
The shape bullet and example text as well as the example with the yellow indicator on the left hand side of the button continue to imply that the indicator must be along the shortest side.   It would be helpful to clarify that the calculation is based on the shortest side but that the indicator could be any shape anywhere in the component as long as it's 2px thick for the size of the minimum area.  So and example of a square might be useful or a horizontal line might make that fact more clear in addition to the current example.

**Comment by alastc** [comment]
Hi @mraccess77, I've added this near the top of the min-area section:

> The Success Criterion defines a minimum area using a calculation for perimeter, and a secondary minimum based on the shortest side of the control. **It does not restrict where the indicators are placed, it is just providing two methods to calculate a minimum area.**

@mbgower, did the "and no part of the contributing area is thinner..." help?

**Comment by alastc** [comment]
Proposed response:

<hr>

The suggested update would rule-out some indicators which meet the intended meaning. I.e. an indicator with a small part that is thinner than 2px would fail, such as a crescent shape.

The change for #1806 (now merged) should also help to address this issue.

**Comment by mbgower** [comment]
>@mbgower, did the "and no part of the contributing area is thinner..." help?

I don't find the bullet name "shape" really aligning with the existing wording or with your variation. It took awhile to parse your update but I now get what you're talking about (the whole crescent shape being dismissed because part of it is less than 2px wide)

Another possible variation:
Shape: The parts of a shape no thinner than 2 CSS pixels, covering an area equivalent to a 4 CSS pixel thick line along the shortest side of a minimum bounding box of the unfocused component.

BTW, the "area equivalent to" is fairly important to clarify it doesn't have to be a 4px blob

**Comment by alastc** [comment]
In the meeting today we agree to implement #2070, which is our attempt to simplify the SC text more. I think that closes this issue, but please re-open if you disagree.
